src/style.css

```diff
@@ -1 +1,73 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #3D4E61;
+  --border-color: #E1E7ED;
+}
+
+.search-bar {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  display: flex;
+  align-items: center;
+  border: 1px solid var(--border-color);
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  border-radius: 4px;
+}
+
+.search-bar--small {
+  height: 42px;
+}
+
+.search-bar__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #fff;
+  border-radius: 5px;
+  transition: var(--border-color) 0.3s;
+  padding-left: 62px;
+  font-size: 16px;
+  color: var(--main-color);
+  background: url('images/Search.svg') no-repeat 26px center / 20px 20px;
+  position: absolute;
+}
+
+::placeholder {
+  color: var(--main-color);
+}
+
+.search-bar__input-small {
+  padding-left: 33px;
+  font-size: 14px;
+  background: url('images/Search.svg') no-repeat 13px center / 11px 11px;
+}
+
+.search-bar:hover {
+  border-radius: 4px;
+  border: 1px solid var(--border-color);
+  background: #FFF;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+}
+
+.search-bar__input:focus {
+  border-radius: 3px;
+  border: 1px solid var(--border-color);
+  text-shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input-small:focus {
```

Comment:
I see there is no such property in figma, in the left sidebar (for some reason), but typed text should be bold in the big input as well. To be able to make it bold also add `Avenir-Heavy` font